File: eRestoran/eRestoran.Model/Uplata.cs
using System;

namespace eRestoran.Model
{
    public class Uplata
    {
        public int Id { get; set; }
        public double Iznos { get; set; }
        public string BrojTransakcije { get; set; }
        public DateTime DatumTransakcije { get; set; }
        public int KorisnikId { get; set; }
    }
}

File: eRestoran/eRestoran.Mobile/lib/models/Uplata.dart
import 'dart:convert';

class Uplata {
  late int? KorisnikId;
  final double Iznos;
  final String BrojTransakcije;
  final DateTime DatumTransakcije;

  Uplata(
      {this.KorisnikId,
      required this.Iznos,
      required this.BrojTransakcije,
      required this.DatumTransakcije});

  Map<String, dynamic> toJson() => {
        "KorisnikId": KorisnikId,
        "Iznos": Iznos,
        "BrojTransakcije": BrojTransakcije,
        "DatumTransakcije": DatumTransakcije.toString(),
      };
}
